ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml

```diff
@@ -90,15 +90,17 @@ tests:
     name: ossm-github-simple-job
 - as: update-istio-builder-image
   commands: |
-    export NEW_IMAGE=$(curl -sL https://raw.githubusercontent.com/istio/common-files/master/files/common/scripts/setup_env.sh | grep IMAGE_VERSION= | cut -d= -f2)
+    export NEW_IMAGE_MASTER=$(curl -sL https://raw.githubusercontent.com/istio/common-files/master/files/common/scripts/setup_env.sh | grep IMAGE_VERSION= | cut -d= -f2)
+    export NEW_IMAGE_1_22=$(curl -sL https://raw.githubusercontent.com/istio/common-files/release-1.22/files/common/scripts/setup_env.sh | grep IMAGE_VERSION= | cut -d= -f2)
     ./tools/automator-main.sh \
     --org=openshift \
     --repo=release \
     --branch=master \
     --token-path=/creds-github/token \
     '--title=Update maistra/istio builder image' \
     --modifier=update_istio_builder_image_master \
-    '--cmd=sed -i "s|\(gcr.io/istio-testing/build-tools\):.*|\1:$NEW_IMAGE|" core-services/image-mirroring/_config.yaml' \
+    '--cmd=sed -i "s|\(gcr.io/istio-testing/build-tools\):master.*|\1:$NEW_IMAGE_MASTER|" core-services/image-mirroring/_config.yaml' \
+    '--cmd=sed -i "s|\(gcr.io/istio-testing/build-tools\):release-1\.22.*|\1:$NEW_IMAGE_1_22|" core-services/image-mirroring/_config.yaml' \
```

Comment:
will this work? two `--cmd` flags? Will it ignore the first and only consider the last one?

I think you will need to merge the two sed commands into a single sed.

Reply:
or two seds separated by `&&` i.e., one single `--cmd` flag that runs multiple commands separated by `&&`

Reply:
Not sure, better to use just one. Fixing. Thanks